src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+h2 {
```

Comment:
Identical styles for tags/classes can be separated by commas to avoid repeating the same code.